checklists/silver.md

```diff
@@ -1,49 +1,35 @@
 ## Silver Certification
 
-**BOLD text is different from Bronze**
+- [ ] **All items from Bronze**
 
 > TODO: Insert example Silver cookie-cutter here.
 
 ### Documentation
 
-- [ ] **Background/significance of program.**
-- [ ] Description of what program does is listed.
-- [ ] **A tutorial that showcases the program's usage. If multiple usages, examples of those are shown also.**
-- [ ] Description of **all** input parameters.
-- [ ] Description of outputs.
-    - Advanced usage
-    - Document functions intended to be used by users.
+- [ ] Background/significance of program
+- [ ] A tutorial that showcases the program's typical intended usage
+- [ ] Advanced usage is described
+- [ ] Thorough description of required and optional input parameters
+- [ ] Document public functions (docstring / help coverage ≥ 20%)
 
 
 ### Infrastructure
 
-- [ ] Readme is present.
-- [ ] DOI (Open Science Framework)
-- [ ] License is present.
-- [ ] Issues are enabled.
-- [ ] **Issue template(s) available (information requested by developers).**
+- [ ] DOI points to latest version
+- [ ] Issue template(s) available (information requested by developers)
 
 
 ### Code
 
-- [ ] Installation instructions available.
-- [ ] No excessive files included (gitignore).
-- [ ] **Adheres to a style guide (i.e., Python and Flake8)**
-- [ ] Portability guide
-    - [ ] Application/Script: **Containerization** (Dockerfile and/or Singularity recipe)
-    - [ ] Utility library: Up to user?
-
+- [ ] No excessive files included (gitignore)
```

Comment:
```suggestion
- [ ] No excessive filesystem files included (e.g., `.swp`, `.DS_STORE`, etc... cam be controlled with a `.gitignore` for GitHub projects)
```

Reply:
I agree with this suggestion, except I think the spirit of the original item also excludes things like unnecessary data and configuration files